packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -75,11 +80,55 @@ export class PayPalHandler extends StripeWebhookHandler {
       throw error.unknownCustomer(uid);
     }
 
+    const { token } = request.payload as Record<string, string>;
+    const currentBillingAgreement = this.stripeHelper.getCustomerPaypalAgreement(
+      customer
+    );
+
+    if (!token && !currentBillingAgreement) {
+      throw error.missingPaypalPaymentToken();
+    }
```

Comment:
This isn't quite the check needed, as it will miss someone with a paypal-funded subscription active that happened to have cancelled their billing agreement. That case will require a different error, informing them that they must fix their PayPal funding for their currently active subscription before adding more subscriptions.